### ISSUE TITLE
Support easy Camunda Cloud configuration

### ISSUE THREAD

**Issue by ArtunSubasi** [zeebe-io/spring-zeebe#81]
The latest version 0.7.1 doesn't seem to support Camunda Cloud out of the box. According to the readme, it is only possible to configure `zeebe.client.broker.contactPoint`. The configuration of a client id and a client secret is not possible.

The #67 should make it possible to add a credentials provider. Nevertheless, it would nice to support Camunda Cloud with less effort. For example:

```yaml
zeebe:
  client:
    broker:
      contactPoint: "Camunda Cloud Contact Point"
    id: "Camunda Cloud Client ID"
    secret: "Camunda Cloud Client Secret"
```

**Comment by salaboy** [comment]
@ArtunSubasi that sounds like a great community contribution, I don’t mind helping, reviewing and testing a PR if you create one. We can definitely create a release before the end of the year if we have that fix in place . I totally agree with you on the fact that it should be just those three properties to get it working with cloud

**Comment by salaboy** [comment]
@ArtunSubasi do you think that you can give it a go? If you do, please share your fork so we can work together on it.

**Comment by ArtunSubasi** [comment]
Sure, I'll give it a go.

**Comment by ArtunSubasi** [comment]
Please have a look at my PR https://github.com/zeebe-io/spring-zeebe/pull/82

**Comment by jwulf** [comment]
Thanks for this @ArtunSubasi - you're a champion!

**Comment by berndruecker** [comment]
I totally agree that we should make it still a bit easier - I want to keep this issue open - as the PR is a great start - but does not solve the core issue

**Comment by ArtunSubasi** [comment]
Feel free to check out the unused branch that I mentioned in the PR: https://github.com/zeebe-io/spring-zeebe/compare/master...ArtunSubasi:security-configuration

**Comment by berndruecker** [comment]
Sorry - I overlooked that. That looks basically good to me (I did a quick and dirty version here https://github.com/berndruecker/flowing-retail/blob/master/zeebe/java/checkout/src/main/java/io/flowing/retail/checkout/ZeebeClientConfiguration.java#L15). 

So the discussion is

> Then I found out that the Camunda Cloud connections work out-of-the-box using environment variables. Therefore, I don't think it is a good idea to add additional support for the application properties. Since the credentials contain confidential information, they must be encrypted or injected using the environment variables anyway.

Any opinions?

My personal opinion is that it would still be valuable to have it via Spring Boot - which also allows you to easily set environment variables. It is simply more aligned with the Spring Boot way - or WDYT? 

And I am the only one that likes to put some defaults simply in the application.yaml and get going to play around locally? Environment variables are an obstacle in that worfklow. WDYT?
